CHANGES.txt

```diff
@@ -15,6 +15,16 @@ Coming in build 307, as yet unreleased
 --------------------------------------
 
 ### pywin32
+* Removed the following unused error names and aliases. (#2269, @Avasam)
+  Here's their replacements if you were using them:
+    * `win32com.universal.com_error` --> `pythoncom.com_error`
```

Comment:
I think we should soften this message a little - IIUC we don't expect any breakage here because strings as exceptions simply don't work, but a quick read implies something like "if you were catching these exceptions, here are the new names you should catch"

For public symbols which were actually errors I see no reason to stop exporting them.

Reply:
Certainly! I'll think how I wanna reword this.